src/ContactPage/Contact.scss

```diff
@@ -2,45 +2,51 @@
 @import "../styles/layout";
 
 .contact-section {
+  background-image: url(../../assets/letters.jpg);
+  
+  
+  height:calc(100vh - 60px);
+  background-size: cover;
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 
   &__border {
     padding: 2% 0;
-    background: rgba(42, 71, 110, 0.918);
     width: 100%;
+    background-color: grey;
+    opacity: 0.9;
+    margin-top: 8%;
   }
   &__title {
     font-family: $base-font-family;
-    margin: 0 5%;
-    margin-bottom: 10px;
-    color: rgb(212, 212, 212);
+    margin: 2% 5% 3% 5%;
+    color: #292828;
     @media (max-width: $mobile-screen) {
       font-size: 20px;
     }
   }
 
   &__form {
-    margin: 0 5%;
+    margin: 0 5% 4%;
     display: flex;
```

Comment:
This isn't a comment on changes you made here. But i think for mobile it would be better for the fields and error fields to stack and everything to go 100% width. 

I would make a media query and then do the above, it's not really too usable on mobile right now